artemis-server/src/main/java/org/apache/activemq/artemis/core/server/Queue.java

```diff
@@ -389,6 +389,8 @@ default int retryMessages(Filter filter, Integer expectedHits) throws Exception
 
    boolean hasMatchingConsumer(Message message);
 
```

Comment:
If you're using this outside of ring, I would rename this to something else...


getRawMessageCount (terrible name I know.. but anything removing the "ring" term)

Reply:
agree, I was leaving it unchanged but calling it what it is PendingMessageCount is best, sorted. thanks.